docs/rules/dom-elements-no-danger-with-children.md

```diff
@@ -0,0 +1,39 @@
+# Prevent problem with children and props.dangerouslySetInnerHTML (dom-elements-no-danger-with-children)
+
+This rule helps prevent problems caused by using children and the dangerouslySetInnerHTML prop at the same time
```

Comment:
This sentence needs ending punctuation.

Also, it might be worth mentioning that React will issue a warning for this case.